wiki/Community/osu!monthly/en.md

```diff
@@ -0,0 +1,53 @@
+# osu!monthly
+
+**osu!monthly** is a monthly newsletter managed by the [osu!monthly Team](/wiki/People/osu!monthly_Team), with the goal of summarising the various community and official events, news, announcements, etc. that occur every month. The project has been led by several other groups of community members in the past, with the goal always being the same: inform the average player about the previous and current events in order to shed light over the entire osu! community.
+
+## Becoming an osu!monthly contributor
+
+osu!monthly is a community effort with many people contributing to its creation. Anyone that wants to help or contribute, in ways such as writing, sharing opinions and thoughts, or providing feedback, is always welcome to do so. The [osu!monthly Team](/wiki/People/osu!monthly_Team) consists of such contributors who consistently help out with the project.
+
+All discussion and project organisation takes place in the `#osu-news` channel of the [osu!dev Discord server](/wiki/Community/osu!dev_Discord_server).
```

Comment:
the whole section is oddly verbose and says a lot about nothing -- I've read it until the end and still not seen anything on how to become a contributor. maybe what you wanted instead was a short sentence or two pointing at the Discord channel only?

Reply:
true, perhaps this last sentence would fit better in the lead paragraph instead of having this entire section